File: hospital-Management-System-Backend/hospital-Management-System-Backend/src/main/java/com/project/hospital_Management_System_Backend/dtos/CompanyDto.java
package com.project.hospital_Management_System_Backend.dtos;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Set;

@NoArgsConstructor
@Data
public class CompanyDto {
    private Long id;

    private String name;

    private String description;

    private Set<AnnouncementsDto> announcements;

    private Set<BasicEmployeeDto> employees;

    private Set<BasicPatientDto> patients;

    private Set<AppointmentsDto> appointments;

    private Set<PrescriptionsDto> prescriptions;

    private Set<ProceduresDto> procedures;
}

File: frontend/hospital-Management-System-frontend/src/app/models/CompanyDTO.ts
import AnnouncementDTO from "./AnnouncementDTO";
import AppointmentDTO from "./AppointmentDTO";
import BasicEmployeeDTO from "./BasicEmployeeDTO";
import BasicPatientDTO from "./BasicPatientDTO";
import PrescriptionDTO from "./PrescriptionDTO";
import ProcedureDTO from "./ProcedureDTO";

export default interface CompanyDTO{
    id:number,
    name:string,
    description:string,
    announcements:AnnouncementDTO[],
    employees:BasicEmployeeDTO[],
    patients:BasicPatientDTO[],
    appointments:AppointmentDTO[],
    prescriptions:PrescriptionDTO[],
    procedures:ProcedureDTO[]
}